packages/core/src/application.ts

```diff
@@ -76,19 +79,44 @@ export class Application extends Context {
    * use the app.server function instead.
    * ```ts
    * app.servers([
-   *  new RestServer(app, options),
-   *  new GRPCServer(app, options),
+   *  RestServer,
+   *  GRPCServer,
    * ]);
    * // Creates a binding for "servers.RestServer" and a binding for
    * // "servers.GRPCServer";
    * ```
    *
-   * @param {Array<Server>} servers An array of instantiated Server instances.
-   * @returns {Binding}
+   * @param {Constructor<Server>[]} ctors An array of Server constructors.
+   * @memberof Application
+   */
+  public servers<T extends Server>(ctors: Constructor<T>[]) {
+    ctors.map(ctor => this.server(ctor));
+  }
+
+  /**
+   * Retrieve the singleton instance for a bound constructor.
+   *
+   * @template T
+   * @param {Constructor<T>} ctor The constructor that was used to make the
+   * binding.
+   * @returns {Promise<T>}
+   * @memberof Application
+   */
+  public async getServer<T extends Server>(ctor: Constructor<T>): Promise<T> {
```

Comment:
Since it's possible to provide a custom server name that does not match the server class name, I think this method should allow callers to provide a server name too, e.g.

```ts
app.getServer<RestServer>('v1API')
```

Thoughts?